app/js/components/manageApps/SingleAddon.jsx

```diff
@@ -0,0 +1,79 @@
+// /*
+//  * This Source Code Form is subject to the terms of the Mozilla Public License,
+//  * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+//  * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+//  * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+//  *
+//  * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+//  * graphic logo is a trademark of OpenMRS Inc.
+//  */
+import React from 'react';
+import { Link } from 'react-router';
+
+export default class SingleAddon extends React.Component{
+    constructor(props){
+        super(props);
+    }
+
+    render(){
+        return(
+            <tr key={this.props.key}>
+                <td onClick={() => openPage(this.props.app.appDetails)}>
```

Comment:
nitpick: You might want to destructure your props - `const { app } = this.props`. Hence you can access the appDetails via `app.appDetails` instead of the verbose `this.props.app.appDetails`